rescan.go

```diff
@@ -482,24 +444,28 @@ func rescan(chain ChainSource, options ...RescanOption) error {
 	//
 	// TODO(wilmer): refactor this and handleBlockDisconnected into their
 	// own methods.
-	handleBlockConnected := func(ntfn *blockntfns.Connected) error {
+	handleBlockConnected := func(ntfn *blockntfns.Connected, isRetry bool) error {
+		// If we have any blocks to retry and the notification doesn't
+		// correspond to the current block being retried, we'll queue it
+		// up to retry after.
+		newHeader := ntfn.Header()
+		if len(retryBlocks) > 0 {
+			retryHeader := retryBlocks[0].Header()
+			if retryHeader.BlockHash() != newHeader.BlockHash() {
+				log.Debug("Stashing", ntfn)
```

Comment:
Reminder to remove, or improve this log message before merge.

Reply:
Log in its full form is actually descriptive if you look at the notification's string method.